docs/data-sources/postgresql_credential.md

```diff
@@ -4,7 +4,7 @@ page_title: "stackit_postgresql_credential Data Source - stackit"
 subcategory: ""
 description: |-
   PostgreSQL credential data source schema. Must have a region specified in the provider configuration.
-  !> The STACKIT PostgreSQL service has reached its end of support on June 30th 2024. Resources of this type have stopped working since then. Use stackitpostgresflexuser instead. For more details, check https://docs.stackit.cloud/stackit/en/bring-your-data-to-stackit-postgresql-flex-138347648.html
+  !> The STACKIT PostgreSQL service has reached its end of support on June 30th 2024. Resources of this type have stopped working since then. Use stackit_postgresflex_user instead. For more details, check
```

Comment:
Not sure why these are here but they are always there when I generate the docs, and only update the files metadata so the content is unchanged

Reply:
This is really weird, not sure what is happening. But anyway, changing the metadata description shouldn't be a problem